src/scielo/bin/xml/prodtools/processing/sps_pkgmaker.py

```diff
@@ -280,7 +280,14 @@ def __init__(self, pkg_path, output_path, optimise=True):
         self.source_folder = workarea.MultiDocsPackageFolder(pkg_path)
 
         # outputs
-        self.output_folder = workarea.MultiDocsPackageOuputs(output_path)
+        try:
+            package_name = os.path.splitext(os.path.basename(pkg_path))[0]
+        except (IndexError, TypeError):
+            package_name = None
+
+        self.output_folder = workarea.MultiDocsPackageOuputs(
+            output_path, package_name=package_name
+        )
```

Comment:
@joffilyfe o nome `scielo_package` é padrão tanto quanto `pmc_package`, `errors`, entre outros e não deveriam ser alterado, quando PackageMaker é instanciado para gerar o pacote XML no XPM.
Mas se for seguir esta abordagem, sugiro que `package_name` seja um parâmetro opcional do `PackageMaker.__init__()` ao invés de criar dentro dele, desta forma ao instancia PackageMaker no contexto do XC, passa-se o valor para `package_name`.

Reply:
Obrigado pela observação @robertatakenaka. Talvez devêssemos escrever testes para que esse comportamento seja preservado.

De toda forma, movi a decisão de utilização do nome do pacote para a classe cliente do `PackageMaker`, a alteração foi feita em 0dc898720abbbd1a398c0f80b55020535da3bea3.